source/common/upstream/ring_hash_lb.h

```diff
@@ -10,6 +10,8 @@
 namespace Envoy {
 namespace Upstream {
 
+using HashFunction = envoy::api::v2::Cluster_RingHashLbConfig_HashFunction;
```

Comment:
Please remove, we don't do this in header files especially.

Reply:
ok, let me move it to the .cc file, so that it's not publicly available.